community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
```

Comment:
It applies to the `APKBUILD`s themselves: contributors are people who contributed to this `APKBUILD`, maintainer is the person that is maintaining the `APKBUILD` (e.g. fixing things, updating it when newer upstream versions are released, etc.). Does that make sense?